### PR TITLE
Pretty WebSocket status messages

### LINKED ISSUE #70: Pretty WebSocket status messages
* Display info about establishing connection/connection is closed
* Colorful cursors for users

### PR DESCRIPTION
**Pull request by ghost**
* Alerts for Websocket connection status

image
image

* Colorful cursors (colors are not synchronized)

image

Resolves #70

### CODE REVIEW

**Comment by katzper01** [comment]
ux vision: maybe lets remove the "Clients connected:..." stringi in main page and instead place some squares (or circles?) with color matching cursors and to not loose information about ids (mainly for debugging), let the Clients id appear on hover as a tooltip, something like that?

**Comment by katzper01** [comment]
and overall - i really like the design!!

**Review by katzper-michno — APPROVED** [review]

File: src/components/editor/latex-textarea/cursors-decorations.ts [hunk 1]
```diff
@@ -1,29 +1,31 @@
 import { MutableRefObject, useEffect, useRef } from 'react';
+
 import { Monaco } from '@monaco-editor/react';
 import { MonacoContentManager } from './MonacoContentManager';
 import { editor } from 'monaco-editor';
 import styles from './LatexTextArea.module.less';
 
 export const useCursorsDecorations = (
   clientId: string,
+  clientsCmap: Map<string, string>,
```
[the remark this answers is not in the document]
Comment (hubzie) [reply src/components/editor/latex-textarea/cursors-decorations.ts:10]: It is from "clients color map"

**Comment by hubzie** [comment]
Prettier user indicators
a
b

File: src/components/editor/latex-textarea/cursors-decorations.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (katzper01) [reply src/components/editor/latex-textarea/cursors-decorations.ts:10]: then use `Colors` instead of `C`

**Comment by katzper01** [comment]
> Prettier user indicators a b

nice

**Pull request merged**